src/domain/profile/events/__tests__/__snapshots__/ProfileEvents.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Renders snapshot correctly 1`] = `<ProfileNoEvent />`;
```

Comment:
This doesn't actually render.. and that might be what we want in this case?

Reply:
Could you elaborate? I am able to run the test on my machine.